### PR TITLE
data: Add a new platform data 'prices'

### PR DESCRIPTION
**Pull request by hajimehoshi**

### CODE REVIEW

File: prices.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "4": "$1.04",
+    "5": "$1.05",
+    "6": "$1.06",
+    "7": "$1.07",
+    "8": "$1.08",
+    "9": "$1.09",
+    "10": "$1.10",
+    "11": "$1.11",
+    "12": "$1.12"
+}
```
Comment (wasedaigo) [inline prices.json]: This will be more like, platform does not know about the int ID.
{
    "com.rpgsnack.bluebird.ads_removal": "$1.01"
}

Comment (hajimehoshi) [reply prices.json]: Huh? I think `data.ShopProduct` has IDs as integer values.

Comment (wasedaigo) [reply prices.json]: This data is associated with the platform domain. Which mean they have no knowledge about ShopProduct defined in game.

Comment (hajimehoshi) [reply prices.json]: Done

**Pull request merged**